pystorm/bolt.py

```diff
@@ -50,6 +58,7 @@ class Bolt(Component):
         from pystorm.bolt import Bolt
 
         class SentenceSplitterBolt(Bolt):
+            output_fields = ['word']
```

Comment:
Just realized this will override the property instead of using the setter.  Seems like I'll need a metaclass and/or descriptor if I want this to work as intended.